azure-mgmt-policyinsights/azure/mgmt/policyinsights/models/policy_events_query_results.py

```diff
@@ -0,0 +1,42 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+from msrest.serialization import Model
+
+
+class PolicyEventsQueryResults(Model):
+    """Query results.
+
+    :param odatacontext: OData context string; used by OData clients to
+     resolve type information based on metadata.
+    :type odatacontext: str
+    :param odatacount: OData entity count; represents the number of policy
+     event records returned.
+    :type odatacount: int
+    :param value: Query results.
+    :type value: list[~azure.mgmt.policyinsights.models.PolicyEvent]
+    """
+
+    _validation = {
+        'odatacount': {'minimum': 0},
+    }
+
+    _attribute_map = {
+        'odatacontext': {'key': '@odata\\.context', 'type': 'str'},
+        'odatacount': {'key': '@odata\\.count', 'type': 'int'},
+        'value': {'key': 'value', 'type': '[PolicyEvent]'},
+    }
+
+    def __init__(self, **kwargs):
+        super(PolicyEventsQueryResults, self).__init__(**kwargs)
+        self.odatacontext = kwargs.get('odatacontext', None)
```

Comment:
@bulentelmaci Do you need `odatacontext` and `odatacount` here? If not, this API should be declared as producing a list (that's the case, you produce a list of PolicyEvent). This makes better SDK, and it's just adding a `x-ms-pageable` extension in the Swagger.
Don't be fooled by the name of the extension, this does *NOT* require the call to be pageable. This is an odd choose to say "the result is a list". Pageable a sub-set of "result is a list".